lib/usecases/environment/GetAllEnvironmentsUseCase.js

```diff
@@ -85,10 +85,17 @@ class GetAllEnvironmentsUseCase {
                 currentStatus: currentStatusExpression,
                 statusHistory,
                 runNumbers: runNumbersExpression,
+                created,
             } = filter;
```

Comment:
I would rename it, e.g. `createdFilter`, `createdExperssion`

Reply:
For using 'expression' in the name, as far as I can see from the code, the convention is to use it when it is a parseable value. Such as a list of runNumbers or ids as a string separated by a comma. 
If we are looking at `GetAllRunsUseCase` or `GetAllLogsUseCase`, you will see the convention for naming filters from/to is the same as this one: `created`, `o2start`, `o2stop`, etc.

Reply:
ok